vortex-datafusion/src/converter.rs

```diff
@@ -0,0 +1,14 @@
+use std::fmt::Debug;
+
+use datafusion_common::stats::Precision;
+use vortex_array::stats;
+
+pub fn directional_bound_to_df_precision<T: Debug + Clone + Eq + PartialOrd>(
```

Comment:
Hmm, let's add a TODO to move all datafusion conversion logic into this crate. Instead of feature flagging in other crates.